### ISSUE TITLE
Clarifying the licensing of all art assets

### ISSUE THREAD

**Issue by akien-mga** [reduz/jetpaca#1]
Here comes the usual boring issue, but it's really necessary to have a sane basis to work on, while the repo is "untouched".
## Code license

**TL;DR:** _It's good._

The repo was pushed currently with the MIT license in a LICENSE.md file. Typically the MIT is used for code (for a Godot game I think we can extend the concept of "code" not only to the scripts, but also the scene files and any text-based resources - the rest would be "asssets").

The license file mentions that the copyright goes to Juan Linietsky (@reduz) and Ariel Manzur (@punto-), and you've told me privately that you own the full rights to open source Jetpaca, so I guess that means this part is covered.

The only thing that could be improved here is to put a short version of the MIT header in each script file so that it's clear that the license applies to those files. I'll make a PR that adds just that. _Edit:_ Done.
## Assets license

**TL;DR:** _We need an asset-specific license (typically Creative Commons), and a list of names of all artists who worked on those assets for attribution._

As specified above, the assets would be anything that is not code: textures, sound effects, music, fonts, etc. I don't know _a priori_ of any incompatibility to use the MIT license to cover the assets, but as it's written as a code-specific license, I believe it would be better to use a Creative Commons license (e.g. CC BY 4.0 or CC BY-SA 4.0).

The tricky part is that those licenses (as well as the MIT actually) require proper attribution of the assets to the artists. So that's the part where we would need some feedback to be able to put names in e.g. a CREDITS.md file so that this part is covered.

I take the assumption that @reduz and @punto- have the full right to set the license for those assets, as that's what @reduz told me and noone has any way to verify it anyway (and I don't have any reason to distrust it either).

Nevertheless, I think we _might_ still be required to give the names of the actual artists who designed the assets - I might be wrong, but at least if we do it we probably have it covered. I think it would be a huge work to take all files separately and give the name of their author, so I would propose to do give attribution "globally" to all assets of the current repo.

To do that, I'd propose that @reduz and @punto- just list the names of every artist that worked on the files in the `art/`, `music/` and `sound/` directories. I would then make a PR with a CREDITS.md file that lists all those files by name, and gives for _all of them_ globally the chosen license and attribution to all the names that were given. WDYT? I think it would be a good enough basis to go on and consider that everything can be assumed fully open source and complying with the chosen license.

**Comment by akien-mga** [comment]
Ping @reduz and @punto-, see my TL;DR about the art assets:

> We need an asset-specific license (typically Creative Commons), and a list of names of all artists who worked on those assets for attribution.

**Comment by reduz** [comment]
cc-by should be fine IMO
Sole artist is Fernando Calabro, though he was being paid so we own the assets

**Comment by akien-mga** [comment]
Thanks, can you add this information to the README.md?

Something like (I'm not a lawyer so I might be wrong in the copyright stuff, but that's how I understand it from what you said):

```
# Licensing

## Code

All scripts and scene files are distributed under the [MIT license](LICENSE.md).  
Copyright held by Juan Linietsky and Ariel Manzur.

## Assets

All art assets (files in ``art/``, ``music/`` and ``sound/``) are distributed under the [Creative Commons Attribution 4.0 International](http://creativecommons.org/licenses/by/4.0/) license.  
Artist: Fernando Calabro  
Copyright held by Juan Linietsky and Ariel Manzur.
```

Then I'll merge this commit in Kobuge's repo and consider that all licensing stuff has been settled.
